backend/migrations/20231010154028_local_image.down.sql

```diff
@@ -0,0 +1 @@
+-- Add down migration script here
```

Comment:
Missing?

Reply:
nothing usefull to do here... we can not just invent image urls and flickr ids